modules/ROOT/pages/redelivery-policy.adoc

```diff
@@ -0,0 +1,62 @@
+= Redelivery Policy
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, flow, redelivery, policy
+
+A redelivery policy is a filter that helps you conserve resources by limiting the number of times the Mule runtime engine (Mule) executes messages that generate an Errors. You can add a redelivery policy to any source in a flow.
```

Comment:
probably "Errors" -> "errors"
These prob belong to an "Error" class, but I'm not sure that we need to be that explicit.